tests/apichecks/testdata/exceptions/shortname_pluralization.txt

```diff
@@ -0,0 +1 @@
+[shortname_plural] crd=computeprojectmetadatas.compute.cnrm.cloud.google.com: plural shortName "gcpcomputeprojectmetadatas" appears to have incorrect pluralization of "gcpcomputeprojectmetadata"
```

Comment:
Can we include if it is v1beta1 in this message? That's important for reviewers to decide wether an action can be required.

Reply:
Added affected versions to this entry.